src/pages/Explorer/Components/ExternalLink.svelte

```diff
@@ -0,0 +1,16 @@
+<script>
+    import Svg from 'webkit/ui/Svg/svelte'
+    export let href;
+</script>
+
+<a {href} target="_blank" class="btn row hv-center border">
+    <Svg id="external-link" w="12" />
+</a>
+
+<style>
+    a { 
+        width: 32px;
+        height: 32px;
+        border-radius: 6px;
+    }
```

Comment:
Use `btn-3` class instead of these styles and some classes (see webkit's `Button` story)